src/Components/ComUserTasksInfo.js

```diff
@@ -1,32 +1,19 @@
 import { NavLink } from 'react-router-dom';
 
 export class ComUserTasksInfo extends Component {
-  constructor(props) {
-    super(props);
-    console.log(props.user, 'user');
-    this.state = {
-      info: props.info
-    };
-  }
-
-
-  static getDerivedStateFromProps(nextProps, prevState) {
-    if (prevState.info !== nextProps.info) {
-      return { info: nextProps.info };
-    }
-    return null;
-  }
 
   render() {
+    const { user } = this.props || {};
+    const { info } = this.props || {};
```

Comment:
1) const { user, info } = this.props || {};
2) is there any condition, when props is not exists?